lib/config/presets/internal/default.ts

```diff
@@ -85,6 +85,17 @@ export const presets: Record<string, Preset> = {
     description: 'Require all status checks to pass before any automerging.',
     ignoreTests: false,
   },
+  automergeStableMinor: {
+    description:
+      'Automerge `minor` upgrades for semver stable packages if they pass tests.',
+    packageRules: [
+      {
+        automerge: true,
+        matchCurrentVersion: '>= 1.0.0',
+        matchUpdateTypes: ['minor'],
```

Comment:
```suggestion
        matchUpdateTypes: ['minor', 'patch'],
```

Reply:
hmm, should it be called `automergeStableMinorAndPatch` then?

Reply:
automergeStableNonMajor

Reply:
Am I assuming correctly that the code suggestion above comes from the fact that the current `automergeMinor` also includes automerging for patch updates?

An alternative would be to implement `automergeCaretConstraint` directly and skip the intermediate step. Any thoughts on that?

Reply:
I'm not sure I follow

Reply:
I was trying to understand why you would prefer to include `patch` in the `matchUpdateTypes` :)

Regarding `automergeCaretConstraint`, I was talking about https://github.com/Turbo87/renovate-config/blob/116fdd4791f2a2e6a9f4692c36b779c62947ca62/automergeCaretConstraint.json which I linked in the PR description above.

Reply:
I think people would expect both patch and minor types to be automerged. It's ok to have a short name if the description is accurate